org.eclipse.jdt.debug/model/org/eclipse/jdt/internal/debug/core/model/JDIThread.java

```diff
@@ -1443,6 +1443,8 @@ private boolean handleSuspendForBreakpointInternal(JavaBreakpoint breakpoint) {
 		catch (CoreException e) {
 			e.printStackTrace();
```

Comment:
It is not caused your change, but while we are here, could you please properly log this exception?